docs/gitlab/gitlab-gcp.md

```diff
@@ -30,10 +30,12 @@ kubectl config set-context [your-cluster-context]
 
 ### Crossplane
 Using the newly provisioned cluster:
+
 - Install Crossplane from master channel using the [Crossplane Installation Guide](../install-crossplane.md#master)
-- Obtain [Cloud Provider Credentials](../cloud-providers.md) 
+- Install the GCP stack into Crossplane using the [GCP stack section](../install-crossplane.md#gcp-stack) of the install guide.
+- Obtain [Cloud Provider Credentials](../cloud-providers.md)
 
-#### GCP Provider   
```

Comment:
some whitespace only changes that were introduced by my auto editor settings.  The main change to this file was to add the link to the GCP stack install section just above.